modules/improvedigitalBidAdapter.js

```diff
@@ -6,14 +6,6 @@ import {Renderer} from '../src/Renderer.js';
 import {hasPurpose1Consent} from '../src/utils/gdpr.js';
 import {ortbConverter} from '../libraries/ortbConverter/converter.js';
 import {convertCurrency} from '../libraries/currencyUtils/currency.js';
-/**
- * See https://github.com/prebid/Prebid.js/pull/8827 for details on linting exception
- * ImproveDigital only imports after winning a bid and only if the creative cannot reach top
- * Also see https://github.com/prebid/Prebid.js/issues/11656
- */
-// eslint-disable-next-line no-restricted-imports
-import {loadExternalScript} from '../src/adloader.js';
```

Comment:
Please remove yourself from the approval list on this file

Reply:
done